### ISSUE TITLE
Parent Experiment ID -- Inconsistency for AerChemMIP ssp370SST- experiments

### ISSUE THREAD

**Issue by charliepascoe** [WCRP-CMIP/CMIP6_CVs#706]
I have been taking a look at the ES-DOC experiment "is_initialized_by" field, checking for consistency with the CMIP6-CV "parent experiment id" and I noticed that ES-DOC and the CMIP6-CV have a discrepancy for the following AerChemMIP experiments:
ssp370SST, ssp370SST-lowNTCF, ssp370SST-lowBC, ssp370SST-lowAer, ssp370SST-lowO3, ssp370SST-lowCH4, ssp370SST-ssp126LU

The CMIP6-CV specifies that these experiments should be initialised from ssp370.
The ES-DOC has no parent experiment specified, though there is an ensemble requirement that implies they should be initialised from the end of the historical simulation.

I am not sure that either is right.  
I think the CMIP6-CV should specify these as having "no parent" for consistency with the similar histSST AerChemMIP experiments (histSST, histSST-piNTCF, histSST-1950HC),
and ES-DOC should remove the "historical initialisation" ensemble requirement.  

I took a look at the AerChemMIP GMD paper to see if it would add clarity, but whilst it has clear instructions for how these experiments should be forced I didn't see anything about the initialisation protocol.
https://www.geosci-model-dev.net/10/585/2017/gmd-10-585-2017.pdf

**Comment by charliepascoe** [comment]
There is a similar situation with the AerChemMIP piClim- experiments.
The CMIP6-CV assigns the piControl as the "parent experiment id" for these experiments
but ES-DOC has no parent experiment specified in the "is_initialized_by" field.
Given that these experiments also have prescribed SSTs I think there is a case for changing the CMIP6-CV to "no parent".

**Comment by durack1** [comment]
@taylor13 can you chime in on these, I'm sure you've interacted on this with Bill or JF way back when these details were finalized

**Comment by durack1** [comment]
@wjcollins we likely need some guidance from the AerChemMIP team here - can you please chime in?

**Comment by wjcollins** [comment]
I'm not quite sure what "Parent" means. These are all fixed-SST runs so it isn't crucial how they are initialized, so long as it is the same for all.

**Comment by durack1** [comment]
@wjcollins "parent" is the experiment that a simulation is initialized from, so for historical this is initialized from some state of the piControl, and the SSP simulations are normally initialized from the last year of the historical (2014). If these are fixed SST sims, and start in 2015 (the ssp370* experiments listed above https://github.com/WCRP-CMIP/CMIP6_CVs/issues/706#issue-452029622) then a historical parent might make ore sense, just like the ssp370 and ssp370-lowNTCF simulations

See [CMIP6_experiment_id.html](https://rawgit.com/WCRP-CMIP/CMIP6_CVs/master/src/CMIP6_experiment_id.html) and search for "aerchemmip ssp" to truncate the experiment list to compare and contrast

**Comment by taylor13** [comment]
@wjcollins @charliepascoe   I agree that for many purposes it is not critical how the prescribed SST experiments are initialized.  There are some processes on land with time constants of several years (notably the water budget of the soil and perhaps more important, vegetation changes (in models with dynamic vegetation) and carbon reservoirs (in models with an active BGC component), when initialization can matter.

If I were going to initialize the 7 ssp370SST... simulations, I would initialize them from the end of the same historical simulation used to initialize my ssp370 experiment.  That way all these runs could be compared without worrying about differences in the initial transient in soil moisture and carbon budgets.

Similarly, for 6 of the 7 histSST... AerChemMIP experiments, I would initialize them from the same point in the piControl simulation that my historical simulation was initialized.  The exception would be histSST-1950HC, which I would initialize from the historical simulation at the beginning of 1950.

I note that across all the MIPs, there are apparent inconsistencies in defining the parent for prescribed SST runs, but some of these are explainable in terms of differences in opinion about how important the initial conditions are.   I note the following:

- Just over 1/3 (104) of the CMIP6 experiments are prescribed SST runs.
- 30 of the prescribed SST runs have "no parent" declared in the CMIP6 CV.
- There are 4 types of prescribed SST experiments:
1. "control" prescribed SST experiments (amip, highresSST-present, amiphist, aqua-control, and control-slab) -- can be initialized however preferred ("no parent" is currently specified in the CV, which is correct).
2. experiments with SSTs taken from a particular coupled simulation (piClim-control, piSST, histSST, ssp370SST, and variants of these) -- if a MIP cares about initialization, these simulations should be initialized from the coupled model state at the beginning of the period for which the SSTs are provided.  
3. a variant of the "amip" experiment (or one of the other prescribed SST "control" runs listed in 1 above) -- If a MIP cares about initialization, these simulations should be initialized from the state of the prescribed SST "control" at the beginning of the period for which the SSTs are imposed.
4. LS3MIP experiments???  -- I'm not sure what type these are.

Currently, for all the category 1 simulations, the CMIP6 CV specifies "no parent", which is correct.

For category 2 simulations:
1. For piClim-control and its 30 variants, "piControl" is specified as the parent because these are initialized from the same point in the piControl where the first prescribed SSTs are extracted.
2. For piSST and its 9 variants, "no parent" indicates that CFMIP doesn't think initialization matters.
3. For histSST and its 6 variants, "no parent" indicates that the MIP doesn't think initialization matters.
4. For ssp370SST and its 6 variants, "ssp370" is specified as the parent, indicating these should be initialized from the (beginning of the) ssp370 coupled run. It would be equally correct to have specified that these simulations be initialized from the end of the "historical" coupled simulation, but I suppose the beginning of ssp370 indicates that the SSTs that follow come from that run.

For category 3 simulations:
1. For variants of the "amip" simulation, PAMIP specifies "amip" as the parent of its 16 related simulations, indicating that these simulations should be initialized just like (the beginning of) the "amip" experiment; other MIPs specify "no parent" for 12 other simulations of this kind.
2. For the 5 variants of highresSST-present, "highresSST-present" is specified as the parent, indicating that these simulations should be initialized just like (the beginning of) the "amip" experiment. 
3. For the 10 variants of histSST and aqua-control, "no parent" is specified, indicating that the MIPs don't think initialization matters
4. For volc-pinatubo-slab, "control-slab" is specified as the parent, indicating that this simulation should be initialized from a point in the "control-slab" experiment.

The bottom line for AerChemMIP is that they need to decide whether they care about how the models are initialized and follow the above rules.  In the current CV, AerChemMIP declares:
1. it cares about initialization in all its prescribed SST experiments *except* for histSST and its 6 variants.
2. The piClim-control simulation and its 18 AerChemMIP variants should all be initialized from the same point in the piControl run that marks the beginning of the piClim SSTs.
3.  The ssp370SST simulation and its 6 variants should all be initialized from the beginning of ssp370 coupled simulation (i.e., the end of the "historical" run).

If this is not correct, please let us know.

**Comment by wjcollins** [comment]
That makes sense to initialize histSST experiments as for historical, and to initialize ssp370SST experiments as for ssp370.

**Comment by wjcollins** [comment]
... Except histSST-1950HC of course which should be as hist-1950HC.

**Comment by taylor13** [comment]
O.K. Then lets modify the CV as follows:

For hist1950HC
parent experiment id: historical
parent activity id: CMIP

For histSST-piCH4, histSST-piN2O, histSST-piO3, histSST, histSST-piAer, histSST-piNTCF
parent experiment id: piControl
parent activity id: CMIP

We also *could* change initialization of the ssp370SST simulation and its 6 variants to "historical" (i.e., end of historical), rather than the current ssp370 (i.e., beginning of ssp370), but I don't think this is absolutely necessary.  What do others think?  The usual practice is to indicate with parent id which simulation preceded the experiment of interest (which would be "historical", not "ssp370"), but I'd like to minimize changing the CV (and as noted above the run takes its SSTs from ssp370, as indicated by the experiment_id: ssp370SST).

**Comment by charliepascoe** [comment]
Thanks @taylor13 your description of the thinking behind the initialisation of the SST runs is very helpful.

**Comment by taylor13** [comment]
@wjcollins  -- After further reflection and with your o.k.,  we'd like to make the parent of the ssp370SST variants the same as the parent of ssp370.  So:

for ssp370SST, ssp370SST-lowAer, ssp370SST-lowBC, ssp370SST-lowCH4, ssp370SST-lowNTCF, ssp370SST-lowO3, set:

parent experiment id: historical
parent activity: CMIP

 and, repeating the earlier suggested changes:

For hist1950HC
parent experiment id: historical
parent activity id: CMIP

For histSST-piCH4, histSST-piN2O, histSST-piO3, histSST, histSST-piAer, histSST-piNTCF
parent experiment id: piControl
parent activity id: CMIP

Please let us know if this is o.k. with you.

**Comment by wjcollins** [comment]
Yes, that seems to make the most sense.

**Comment by durack1** [comment]
@wjcollins those changes are now in place. Please review the updated details at [CMIP6_experiment_id.html](https://rawgit.com/WCRP-CMIP/CMIP6_CVs/master/src/CMIP6_experiment_id.html) and let us know of any other tweaks required

@charliepascoe ping

**Comment by charliepascoe** [comment]
The the parent experiment id for histSST-1950HC is still set as "no parent" in the CMIP6-CV, should this be amended to "historical"?

**Comment by durack1** [comment]
@charliepascoe you make a good point. The other AerChemMIP experiments extend from 1850 to 2014 and are initialized from the piControl, I am curious if histSST-1950HC should be initialized from the historical as you note, the 1950 halocarbon concentrations are the key descriptive field from what I can ascertain from the experiment description

**Comment by wjcollins** [comment]
With further thought, the histSST-1950HC should ideally be initialized to histSST at 1950. The analysis will compare histSST and histSST-1950HC, so at Jan 1st 1950 the delta should be zero. I know at least one group has already run this experiment, but I don't know how they intialized it.

**Comment by durack1** [comment]
@wjcollins thanks for clarifying, so edits required are:

```
        "histSST-1950HC":{
            "activity_id":[
                "AerChemMIP"
            ],
            "additional_allowed_model_components":[
                "BGC"
            ],
            "description":"Historical WMGHG concentrations and NTCF emissions, 1950
 halocarbon concentrations",
            "end_year":"2014",
            "experiment":"historical SSTs and historical forcing, but with 1950
halocarbon concentrations. Experiment is initialized from histSST (AerChemMIP)
simulation from January 1950",
            "experiment_id":"histSST-1950HC",
            "min_number_yrs_per_sim":"65",
            "parent_activity_id":[
                "AerChemMIP"
            ],
            "parent_experiment_id":[
                "histSST"
            ],
            "required_model_components":[
                "AGCM",
                "AER",
                "CHEM"
            ],
            "start_year":"1950",
            "sub_experiment_id":[
                "none"
            ],
            "tier":"1"
},
```

Please confirm this is correct and we'll get this updated

**Comment by durack1** [comment]
@wjcollins @charliepascoe I have taken the liberty of assuming my edits https://github.com/WCRP-CMIP/CMIP6_CVs/issues/706#issuecomment-503224221 are correct and these are now implemented. Please take a peek at [CMIP6_experiment_id.html](https://rawgit.com/WCRP-CMIP/CMIP6_CVs/master/src/CMIP6_experiment_id.html) and let me know of any remaining issues

**Comment by wjcollins** [comment]
It has been noticed that ssp370SST-ssp126LU still has the ssp370 parent.
Please change parent to historical, and parent activity to CMIP

**Comment by durack1** [comment]
@wjcollins that change is now in place. Please peruse [CMIP6_experiment_id.html](https://rawgit.com/WCRP-CMIP/CMIP6_CVs/master/src/CMIP6_experiment_id.html) and let us know of any remaining issues
